src/app/Sidebar.js

```diff
@@ -58,10 +57,17 @@ export default class Sidebar extends Component {
         isLoaded: true,
         categories: categories,
         props: result.props,
-        feedPrice: result.feed_price
       });
       this.getFollowing();
     });
+
+    fetch('https://api.coinmarketcap.com/v1/ticker/steem/')
+      .then(res =>
+        res.json().then((json) => {
```

Comment:
Writing as would be better.
```js
fetch('https://api.coinmarketcap.com/v1/ticker/steem/')
      .then(res => res.json())
      .then((json) => {
        const priceUsd = json[0].price_usd;
        this.setState({ price: priceUsd });
      });
```
Also it would better to save price in redux save.